waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/AbstractMetaStore.java

```diff
@@ -152,6 +152,12 @@ public void setLatency(long latency) {
     this.latency = latency;
   }
 
+  public abstract List<String> getMappedDatabases();
+
+  public abstract void setMappedDatabases(List<String> mappedDatabases);
+
+  public abstract boolean shouldHaveNoMappedDatabases();
```

Comment:
I think this doesn't end up in the yml, could you check if it ends up in the json? http://localhost:18000/api/admin/federations/

Reply:
It doesn't (which is good, right?), but `"mappedDatabases":[]` does

Reply:
yes